docs/pbs/index.md

```diff
@@ -115,287 +178,74 @@ Job ID  User     Queue  Nodes NCPUs Finish  RMem(GB)  Mem(GB) CPU(%) Elap(h)
 
+You can obtain additional job details using `qhist -w` for wide output, or customize the output. See `qhist --format=help` for a list of options.
 The following variation will generate a list of jobs that finished with non-zero exit codes to help you identify jobs that failed.
 
-```bash
+```
 qhist -u $USER -r x0
 ```
 
-### `qstat`
-
+##### qstat
 Run this to see the status of all of your own unfinished jobs.
 
-```bash
+```
 qstat -u $USER
 ```
-
 Your output will be similar to what is shown just below. Most column headings are self-explanatory – `NDS` for nodes, `TSK` for tasks, and so on.
 
-In the status `(S)` column, most jobs are either queued `(Q)` or running `(R)`. Sometimes jobs are held `(H)`, which might mean they are dependent on the completion of another job. If you have a job that is held and is not dependent on another job, CISL recommends killing and resubmitting the job.
-
-
-```pre
+In the status (S) column, most jobs are either queued (Q) or running (R). Sometimes jobs are held (H), which might mean they are dependent on the completion of another job. If you have a job that is held and is not dependent on another job, CISL recommends killing and resubmitting the job.
+```
                                                        Req'd  Req'd   Elap
 Job ID         Username Queue   Jobname SessID NDS TSK Memory Time  S Time
 ------         -------- -----   ------- ------ --- --- ------ ----- - ----
 657237.chadmin apatelsm economy ens603   46100 60  216   --   02:30 R 01:24
 657238.chadmin apatelsm regular ens605     --   1   36   --   00:05 H   --
 657466.chadmin apatelsm economy ens701    5189 60  216   --   02:30 R 00:46
 657467.chadmin apatelsm regular ens703     --   1   36   --   00:10 H   --
-
 ```
 
 Following are examples of `qstat` with some other commonly used options and arguments.
 
 Get a long-form summary of the status of an unfinished job.
-
-```bash
+```
 qstat -f jobID
 ```
-!!! warning
-    Use the above command only sparingly; it places a high load on PBS.
+!!! Warning
 
+    Use the above command only sparingly; it places a high load on PBS.
 
 Get a single-line summary of the status of an unfinished or recently completed job (within 72 hours).
 
-```bash
+```
 qstat -x jobID
 ```
-
 Get information about unfinished jobs in a specified execution queue.
 
-
 ```
-qstat queue_name
+qstat <queue_name>
 ```
 See job activity by queue (e.g., pending, running) in terms of numbers of jobs.
 
 ```
 qstat -Q
 ```
-
 Display information for all of your pending, running, and finished jobs.
 
 ```
 qstat -x -u $USER
 ```
-
-Query jobs running on one system by specifying the system as shown here. (Only these options are supported when running qstat in this cross-server mode: `-x`, `-u`,`-w`, `-n`, `-s`)
+Query jobs running on one system by specifying the system as shown here. (Only these options are supported when running qstat in this cross-server mode: `-x, -u,-w, -n, -s`)
```

Comment:
This is also no longer true in the latest version of the cache. It can be removed!